src/components/VerticalCard/VerticalCard.tsx

```diff
@@ -49,50 +53,60 @@ export const VerticalCard = ({
       className={cnb(styles.root, className)}
       {...props}
     >
-      {imageSrc && (
-        <div className={styles.imageWrapper}>
-          <img
-            alt=""
-            src={getProcessedImage(imageSrc, '600x600', imageFocus)}
-            className={styles.image}
-          />
-        </div>
-      )}
-      {heading && (
-        <Heading
-          as={headingLevel}
-          font={headingFont}
-          size={isSmallHeading ? 3 : 4}
-          leading="tight"
-          className={styles.heading}
-        >
-          {(!ctaLabel && (link || href))
-            ? (
-              <CtaLink sbLink={link} href={href} className={styles.headingLink}>
-                {heading}
-              </CtaLink>
-            ) : heading}
-        </Heading>
-      )}
-      {tabColor && (
+      <div className={styles.cardWrapper}>
+        {imageSrc && (
+          <div className={styles.imageWrapper}>
+            <img
+              alt=""
+              src={getProcessedImage(imageSrc, '600x600', imageFocus)}
+              className={styles.image}
+            />
+          </div>
+        )}
+        {heading && (
+          <Heading
+            as={headingLevel}
+            size={isSmallHeading ? 3 : 4}
+            leading="tight"
+            className={cnb(styles.heading(!!tabColor), accentBorderColors[tabColor])}
+          >
+            {(!ctaLabel && (link || href))
+              ? (
+                <CtaLink sbLink={link} href={href} className={styles.headingLink}>
+                  {heading}
+                </CtaLink>
+              ) : heading}
+          </Heading>
+        )}
+        {/* {tabColor && (
```

Comment:
Kerri is updating theme card requirements - so this tab will be moved in an upcoming PR